### ISSUE TITLE
"Show current" seems to crash the card

### ISSUE THREAD

**Issue by sgofferj** [bramkragten/weather-card#58]
I also had the problem with the card disappearing recently and did a bit of experimentation.
It appears, when I select "show current", the card crashes or is not loaded. 

I see the following error in the console:
```
weather-card.js:169
Uncaught (in promise) TypeError: Cannot read property 'state' of undefined
    at HTMLElement.renderCurrent (weather-card.js:169)
    at HTMLElement.render (weather-card.js:151)
    at HTMLElement.update (app.9108bd0a.js:1)
    at HTMLElement.performUpdate (app.9108bd0a.js:1)
    at HTMLElement._enqueueUpdate (app.9108bd0a.js:1)
```
When I disable/omit "show current", the card loads, however it does not display it's name.
The error is the same if I use the locally installed (HACS) version or the hosted one.

Tested with Chrome and firefox on Ubuntu 18.04. HA is the latest docker image running on docker.

It might be related to #54 but I'm not sure, so I opened a separate issue.

**Comment by bramkragten** [comment]
Make sure you have the sun loaded.

**Comment by bramkragten** [comment]
Fixed in v1.4.6

**Comment by sgofferj** [comment]
Confirmed fixed
